Client/Frontend/Browser/Playlist/Cells/PlaylistCellRedacted.swift

```diff
@@ -11,126 +11,27 @@ import DesignSystem
-  
-  private let url: URL?
-  private let scale: CGFloat
-  private let transaction: Transaction
-  private let render: (AsyncImagePhase) -> Content
-  
-  public init(url: URL?, scale: CGFloat = 1) where Content == Image {
-    self.url = url
-    self.scale = scale
-    self.transaction = Transaction()
-    self.render = { $0.image ?? Image("") }
-  }
-
-  init<ContentView: View, PlaceHolder: View>(url: URL, scale: CGFloat = 1, @ViewBuilder content: @escaping (Image) -> ContentView, @ViewBuilder placeholder: @escaping () -> PlaceHolder) where Content == _ConditionalContent<ContentView, PlaceHolder> {
-      
-    self.url = url
-    self.scale = scale
-    self.transaction = Transaction()
-    
-    self.render = { phase -> _ConditionalContent<ContentView, PlaceHolder> in
-      if let image = phase.image {
-        return ViewBuilder.buildEither(first: content(image))
-      } else {
-        return ViewBuilder.buildEither(second: placeholder())
-      }
-    }
-  }
-  
-  public init(url: URL?, scale: CGFloat = 1, transaction: Transaction = Transaction(), @ViewBuilder content: @escaping (AsyncImagePhase) -> Content) {
-    self.url = url
-    self.scale = scale
-    self.transaction = transaction
-    self.render = content
-  }
-
-  public var body: some View {
-    render(loader.phase)
-      .onAppear {
-        loader.load(url, scale: scale, transaction: transaction)
-      }
-      .onChange(of: url) { url in
-        loader.load(url, scale: scale, transaction: transaction)
-      }
-  }
-  
-  private class ImageLoader: ObservableObject {
-    @Published var phase: AsyncImagePhase
-    
-    init() {
-      self.phase = .empty
-    }
-      
-    func load(_ url: URL?, scale: CGFloat, transaction: Transaction) {
-      guard let url = url, !url.absoluteString.isEmpty else {
-        withTransaction(transaction) {
-          self.phase = .empty
-        }
-        return
-      }
-      
-      let session = URLSession(configuration: .ephemeral)
-      Task { @MainActor in
-        do {
-          let (data, _) = try await session.dataRequest(with: url)
-          withTransaction(transaction) {
-            if let image = UIImage(data: data, scale: scale) {
-              self.phase = .success(Image(uiImage: image))
-            } else {
-              self.phase = .empty
-            }
-          }
-        } catch {
-          self.phase = .failure(error)
-        }
-      }
-    }
-  }
-  
-  public enum AsyncImagePhase {
-    case empty
-    case success(Image)
-    case failure(Error)
-
-    public var image: Image? {
-      switch self {
-      case .empty, .failure: return nil
-      case .success(let image): return image
-      }
-    }
-    
-    public var error: Error? {
-      switch self {
-      case .empty, .success: return nil
-      case .failure(let error): return error
-      }
-    }
-  }
-}
-
 private struct PlaylistRedactedHeaderView: View {
+  @Observable var title: String?
+  @Observable var creatorName: String?
```

Comment:
`Observable` means nothing here? Just pass in variables directly

Reply:
Fixed